### PR TITLE
More MO tests

### PR DESCRIPTION
**Pull request by marisademeglio**
Three tests for this subsection: https://w3c.github.io/epub-specs/epub33/rs/#sec-rsconf-rendering-audio

### CODE REVIEW

**Comment by iherman** [comment]
> > the content should be in an OPS directory
> 
> Why are you using OPS, out of curiosity? That's the name of the EPUB 2 structure spec.

I do not remember... but all our tests use the same structure, which is helpful. 

Cc @dlazin

**Comment by mattgarrish** [comment]
> but all our tests use the same structure, which is helpful.

Sure, but that's why you'll find /EPUB directories used in all the post EPUB 2 stuff we did in IDPF. You used to also find a lot of /OEBPS directories, too, which came from the even older Open EBook Publication Structure spec name.

It's kind of weird to use EPUB 2 terminology with EPUB 3.

**Comment by marisademeglio** [comment]
I have used "EPUB" because I like it better :) I didn't know about the requirement to use "OPS". Do you want me to change it?

**Comment by iherman** [comment]
> I have used "EPUB" because I like it better :) I didn't know about the requirement to use "OPS". Do you want me to change it?

I let @dlazin a.k.a. the boss decide it, but I think it would be better.

**Comment by dlazin** [comment]
> > > the content should be in an OPS directory
> > 
> > 
> > Why are you using OPS, out of curiosity? That's the name of the EPUB 2 structure spec.
> 
> I do not remember... but all our tests use the same structure, which is helpful.
> 
> Cc @dlazin

This came from the templates that Dave created; I just cleaned them up for consistency, and wasn't aware that OPS was an EPUB 2 convention; I thought the name was arbitrary and OPS was one of the common choices. (InDesign produces an OEBPS folder, as I'm sure you know.)

We can change it, but I would like to be consistent at any point in time, because that way you can more easily tell what the intentional choices in a given test are relative to the default non-choices. So for now, indeed, please do OPS and otherwise match the template in https://github.com/w3c/epub-tests/tree/main/tests/xx-epub-template, and we can rename all tests to EPUB in a separate PR if you think it's worth doing.

**Comment by dlazin** [comment]
Folder naming aside, Ivan and Marisa mentioned in today's testing call that they both wrote the same test. I have reviewed #118 and am working on #116 right now; does this still conflict? If so, please fix before I review; otherwise let me know and I will try to review tomorrow! Thanks and sorry for the collision.

**Comment by iherman** [comment]
Looking at the tests, it looks like one of the tests in the PR (`mol-rendering-audio`) overlaps with three tests in #116. The difference is their granularity: the test in this PR takes the whole section (i.e., the three _MUST_ statements) in one test, whereas the ones of #116 treat them separately, i.e., a separate test for each of them. Personally, I would favour a smaller granularity; wdyt, @marisademeglio?

The other test is in navigation that do not overlap with neither #116 nor #118 in my view.

**Comment by marisademeglio** [comment]
> Looking at the tests, it looks like one of the tests in the PR (`mol-rendering-audio`) overlaps with three tests in #116. The difference is their granularity: the test in this PR takes the whole section (i.e., the three _MUST_ statements) in one test, whereas the ones of #116 treat them separately, i.e., a separate test for each of them. Personally, I would favour a smaller granularity; wdyt, @marisademeglio?
> 

I could argue it either way; I was following the example of the tests in the previous MO section (timing and sync), which combine several MUSTs into a single test. The test in "rendering audio" can be for the concept of "media clipping", which is what this PR does; or the tests can be for each permutation of media clipping, as in #116.

**Comment by mattgarrish** [comment]
> I thought the name was arbitrary and OPS was one of the common choices.

Sure, you can call the folder whatever you want, or not even bother with one. But for a formal test suite, it's a weird association with an acronym of the past. We're not testing OPS or OEBPS structure. That's why we opted to use "EPUB" as the name for all things 3.0+ -- to break the cycle of revision-based acronyms. It's why the specs are all "EPUB XYZ" now, too, as it didn't help to have a format called EPUB defined by standards with acronyms like OPS and OPF.

I can fix this for you if you want, though, since I'm the one complaining (in a separate PR, of course).

**Comment by dlazin** [comment]
> I can fix this for you if you want, though, since I'm the one complaining (in a separate PR, of course).

Sure, that would be very welcome! I would happily accept a separate PR that renames all OPS folders to EPUB.

**Comment by iherman** [comment]
> > I can fix this for you if you want, though, since I'm the one complaining (in a separate PR, of course).
> 
> Sure, that would be very welcome! I would happily accept a separate PR that renames all OPS folders to EPUB.

I think that is really the issue: I am afraid this boat has sailed. There are about 100 tests in the repo; we should come up with a script that makes the folder name change in all folders; modify the necessary files within the folders; and re-generate the epub files. Doable of course, but I am not sure it is worth at this point...

**Comment by iherman** [comment]
> I could argue it either way; I was following the example of the tests in the previous MO section (timing and sync), which combine several MUSTs into a single test. The test in "rendering audio" can be for the concept of "media clipping", which is what this PR does; or the tests can be for each permutation of media clipping, as in #116.

True enough.

My pragmatic point is that having more tests does not harm; I would just take all of these. I hate throwing out things that we already have. @dlazin ?

**Comment by iherman** [comment]
> > > I can fix this for you if you want, though, since I'm the one complaining (in a separate PR, of course).
> > 
> > 
> > Sure, that would be very welcome! I would happily accept a separate PR that renames all OPS folders to EPUB.
> 
> I think that is really the issue: I am afraid this boat has sailed. There are about 100 tests in the repo; we should come up with a script that makes the folder name change in all folders; modify the necessary files within the folders; and re-generate the epub files. Doable of course, but I am not sure it is worth at this point...

Oops, I just see that @mattgarrish has done this in #119!

I guess this means I would have to change things manually in #116 and #118

**Comment by iherman** [comment]
> True enough.
> 
> My pragmatic point is that having more tests does not harm; I would just take all of these. I hate throwing out things that we already have. @dlazin ?

Thinking about this further... I convinced myself that @marisademeglio is right :-)

I will then remove two tests from #116: namely `mol-audio_no_clipbegin` and `mol-audio_no_clipend`. I will, however, repurpose `mol-audio_exceeding_clipend` because it has shown a possible RS bug already: I will turn into `mol-timing-synchronization_multiple_audio`, reusing the fact that it has a SMIL file that refers, sequentially, to _two_ audio files; this is perfectly valid and should be invisible to the reader.

@dlazin your turn in reviewing this PR...

File: tests/mol-navigation/EPUB/mo/ch1.smil [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<smil xmlns:epub="http://www.idpf.org/2007/ops" xmlns="http://www.w3.org/ns/SMIL" version="3.0">
+  <body epub:textref="../xhtml/ch1.xhtml">
```
Comment (iherman) [inline tests/mol-navigation/EPUB/mo/ch1.smil:2]:
```suggestion
  <body epub:textref="../ch1.xhtml">
```

**Review by iherman — COMMENTED** [review]
epubcheck (the version for EPUB 3.3) reports errors on both files:

```
8:59 tests> epubcheck5 mol-navigation.epub 
Validating using EPUB version 3.3 rules.
ERROR(MED_014): mol-navigation.epub/EPUB/mo/ch1.smil(2,43): A non-empty fragment identifier is required.
ERROR(MED_014): mol-navigation.epub/EPUB/mo/ch2.smil(2,37): A non-empty fragment identifier is required.
ERROR(RSC-007): mol-navigation.epub/EPUB/mo/ch1.smil(2,43): Referenced resource "EPUB/xhtml/ch1.xhtml" could not be found in the EPUB.

Check finished with errors
Messages: 0 fatals / 3 errors / 0 warnings / 0 infos

EPUBCheck completed
```

and 

```
9:07 tests> epubcheck5 mol-navigation.epub     
Validating using EPUB version 3.3 rules.
ERROR(MED_014): mol-navigation.epub/EPUB/mo/ch1.smil(2,37): A non-empty fragment identifier is required.
ERROR(MED_014): mol-navigation.epub/EPUB/mo/ch2.smil(2,37): A non-empty fragment identifier is required.

Check finished with errors
Messages: 0 fatals / 2 errors / 1 warning / 0 infos

EPUBCheck completed
```

The missing file reference is easy (and I submitted an editorial change); for the other issues I _believe_ the issue is that the reference in the smil file's `<body>` element should refer to a fragment in the target xhtml file: ie, and `id` value should be given to, e.g., the `<body>` element of the relevant xhtml file and referred to from the smil file.

**Comment by iherman** [comment]
FWIW, the navigation test runs on the web version of Colibrio as well as on Thorium on the Mac. The rendering audio one runs on Colibrio but the third 'subtest' (`clipEnd` exceeding) does not work on Thorium (the other two sub-tests do work).

The exceeding `clipEnd` stuff seems to be the tricky one... interestingly, this tests use the same trick that I tried, namely using several audio files, and that is where Thorium failed on the earlier tests as well. On the other hand, Thorium actually is o.k. with a 'regular' change of one media file to the other (the separate test I added works well).

**Comment by marisademeglio** [comment]
Thanks for the file reference fix :) 

I was not getting EPUBCheck errors with version 4.2.4 but I've now upgraded to v5 and I see what you see.

However, the error about there being no fragment identifier in the URL, e.g. `<body epub:textref="file.xhtml">` rather than `"file.xhtml#ident"`, does not match what the [spec says for `body`](https://w3c.github.io/epub-specs/epub33/core/#sec-smil-body-elem)

"
The value MUST be a [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string), optionally followed by U+0023 (#) and a [URL-fragment string](https://url.spec.whatwg.org/#url-fragment-string).
"

It does not affect this test but does it seem wrong to you too? I can open an issue over in the appropriate repo. 

Anyway, I am pushing a fix to make EPUBCheck happier, since it doesn't really matter for our purposes here if it's flagging an error when there is none.

**Comment by iherman** [comment]
> However, the error about there being no fragment identifier in the URL, e.g. `<body epub:textref="file.xhtml">` rather than `"file.xhtml#ident"`, does not match what the [spec says for `body`](https://w3c.github.io/epub-specs/epub33/core/#sec-smil-body-elem)
> 
> " The value MUST be a [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string), optionally followed by U+0023 (#) and a [URL-fragment string](https://url.spec.whatwg.org/#url-fragment-string). "

Oops, you are right. I must admit I did not check back with the spec; this may be then a bug in epubcheck5.

I will submit a bug report to epubcheck5 (but ping @rdeltour nevertheless).

File: tests/mol-rendering-audio/EPUB/clip_times_tests.xhtml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8"/>
+        <title>EPUB Media Overlays Tests: Rendering Audio</title>
+        <link rel="stylesheet" type="text/css" href="css/base.css"/>
+    </head>
+    <body>
```
Comment (iherman) [inline tests/mol-rendering-audio/EPUB/clip_times_tests.xhtml:7]:
```suggestion
    <body id="body">
```

File: tests/mol-rendering-audio/EPUB/mo/clip_times_tests.smil [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<smil xmlns:epub="http://www.idpf.org/2007/ops" xmlns="http://www.w3.org/ns/SMIL" version="3.0">
+  <body epub:textref="../clip_times_tests.xhtml">
```
Comment (iherman) [inline tests/mol-rendering-audio/EPUB/mo/clip_times_tests.smil:2]:
```suggestion
  <body epub:textref="../clip_times_tests.xhtml#body">
```

**Review by iherman — CHANGES_REQUESTED** [review]

**Comment by iherman** [comment]
@marisademeglio see https://github.com/w3c/epubcheck/issues/1301

**Comment by dlazin** [comment]
> > I can fix this for you if you want, though, since I'm the one complaining (in a separate PR, of course).
> 
> Sure, that would be very welcome! I would happily accept a separate PR that renames all OPS folders to EPUB.

Matt did this in https://github.com/w3c/epub-tests/pull/119, so you can leave your folder names as EPUB now. Thanks, Matt! And thanks for bringing this to my attention, Marisa.

**Comment by dlazin** [comment]
> > True enough.
> > My pragmatic point is that having more tests does not harm; I would just take all of these. I hate throwing out things that we already have. @dlazin ?
> 
> Thinking about this further... I convinced myself that @marisademeglio is right :-)
> 
> I will then remove two tests from #116: namely `mol-audio_no_clipbegin` and `mol-audio_no_clipend`. I will, however, repurpose `mol-audio_exceeding_clipend` because it has shown a possible RS bug already: I will turn into `mol-timing-synchronization_multiple_audio`, reusing the fact that it has a SMIL file that refers, sequentially, to _two_ audio files; this is perfectly valid and should be invisible to the reader.
> 
> @dlazin your turn in reviewing this PR...

Reviewing now. Just to confirm, we've decided that all the tests in this PR will survive, because Ivan removed some from https://github.com/w3c/epub-tests/pull/116, right? Cool.

File: tests/mol-navigation/EPUB/ch2.xhtml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8"/>
+        <title>Chapter 2</title>
+        <link rel="stylesheet" type="text/css" href="css/base.css"/>
+    </head>
+    <body id="body">
+        <h1 id="mo-1">Chapter 2</h1>
+        <p id="mo-2">This page should play when "Chapter 2" is selected from the table of contents.</p>
```
Comment (dlazin) [inline tests/mol-navigation/EPUB/ch2.xhtml:9]: For clarity, I'd recommend something explicit like "Test passes if the media overlay audio on this page plays when..." (and does the reader need to enable MO on the first page before doing that? If so, please say so.)

**Review by dlazin — APPROVED** [review]
Thank you so much for the tests! Sorry for the delay and all the comments; most of these are just first-time "here is how we do things" changes, not substantive; next time should be much cleaner.

File: tests/mol-navigation/EPUB/css/base.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@charset "utf-8";
+
+/*for the active text element*/
+.-epub-media-overlay-active {
```
[the remark this answers is not in the document]
Comment (marisademeglio) [reply tests/mol-navigation/EPUB/css/base.css:4]: Ok we can make it match across all the tests but I don't like the classnames in the test you linked to. Let's copy the [spec example](https://w3c.github.io/epub-specs/epub33/core/#example-associating-style-information-with-the-currently-playing-epub-content-document). We have found that if we put out documentation with a classname that sounds too nerdy or "official", implementers treat them like magic strings and start supporting those exact classnames. So using `my-active-item` helps to not reinforce this.

File: tests/mol-navigation/EPUB/ch2.xhtml (hunk 4 above)
Comment (marisademeglio) [reply tests/mol-navigation/EPUB/ch2.xhtml:9]: The instructions start on chapter 1, please see that page first and then let me know if it still doesn't make sense.

I will update the wording to say "The test passes if..."

File: tests/mol-rendering-audio/EPUB/package.opf [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System implements Media Overlays' RS Spec "Rendering Audio" section.</dc:description>
+    <dc:identifier id="pub-id">mol-rendering-audio</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Media Overlays Tests: Rendering Audio</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://w3c.github.io/epub-specs/epub33/rs/#sec-rsconf-rendering-audio</meta>
```
[the remark this answers is not in the document]
Comment (marisademeglio) [reply tests/mol-rendering-audio/EPUB/package.opf:14]: Yes I was just focusing on the MUSTs because I thought that was what we needed to tackle first. I can write test for that last SHOULD (will prob be a separate EPUB since that test does not have anything in common with the others).

And when I add the test references to the normative statement, as per step 12, I use the test IDs, but where should these be defined for this situation of having multiple tests in one EPUB?

**Comment by marisademeglio** [comment]
I think that having multiple tests in a single EPUB, as in `mol-rendering-audio` does not work with the test system that has been set up here. The easiest solution is to remove `mol-rendering-audio` and replace with the tests that Ivan had written which test the same things.

**Comment by iherman** [comment]
> I think that having multiple tests in a single EPUB, as in `mol-rendering-audio` does not work with the test system that has been set up here. The easiest solution is to remove `mol-rendering-audio` and replace with the tests that Ivan had written which test the same things.

Well... I have already removed those tests from the respective PR and merge that one. I guess we could find those somewhere in github's memory... (or my mac backups)

@marisademeglio what do you prefer the next action would/should be?

**Comment by marisademeglio** [comment]
I would do whatever is faster :) This week I don't have much time to spend on this, unfortunately.

**Comment by iherman** [comment]
> I would do whatever is faster :) This week I don't have much time to spend on this, unfortunately.

Ok, I have re-instated the three tests for clipend and clipbegin, see #125.

@marisademeglio you can then remove the corresponding test from this PR.

File: tests/mol-rendering-audio/EPUB/package.opf (hunk 6 above)
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-rendering-audio/EPUB/package.opf:14]: In that situation, you would have a single epub but multiple data-tests attributes pointing to the same test, like so:

<p id="first-anchor" data-tests="#mol-rendering-audio">First normative statement.</p>
<p id="second-anchor" data-tests="#mol-rendering-audio">Second normative statement.</p>
<p id="third-anchor" data-tests="#mol-rendering-audio">Third normative statement.</p>

And then mol-rendering-audio.epub would have three dcterms:isReferencedBy:
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#first-anchor</meta>
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#second-anchor</meta>
<meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#third-anchor</meta>

Make sense?

File: tests/mol-navigation/EPUB/css/base.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-navigation/EPUB/css/base.css]: Sure, sounds good.

File: tests/mol-rendering-audio/EPUB/package.opf (hunk 6 above)
[the remark this answers is not in the document]
Comment (marisademeglio) [reply tests/mol-rendering-audio/EPUB/package.opf:14]: And you can't refer to a test below the package level of the test EPUB. Is that right? 

It doesn't matter at this point for me but conceptually I am curious.

**Review by dlazin — COMMENTED** [review]
Hi! Very sorry for the slow reviews here — can you summarize the current state of this PR for me? Is it waiting on my re-review, or do you have changes you need to make first?

(I could probably figure this out myself by careful reading, but I have other PRs to catch up on too.)

Also, attached is a second comment that apparently I wrote weeks ago and neglected to post. I hope it's still relevant!

**Comment by iherman** [comment]
@marisademeglio I wonder where we are with these tests... It would be good to close all outstanding issues and tests before going to CR (which does not mean that we would not need new tests, of course)

**Review by iherman — APPROVED** [review]

**Comment by marisademeglio** [comment]
> @marisademeglio I wonder where we are with these tests... It would be good to close all outstanding issues and tests before going to CR (which does not mean that we would not need new tests, of course)

Of the 4 tests in this PR, we are not using the 3 media clipping tests, so that just leaves `mol-navigation`. I am looking at a [comment](https://github.com/w3c/epub-tests/pull/117#discussion_r812019431) from @dlazin and I gather that 2 things are being requested:

* change to "test passes if..." language: this has already been [done](https://github.com/marisademeglio/epub-tests/blob/409194f4cf1481dc7b20a9df759090d112c852d9/tests/mol-navigation/EPUB/ch2.xhtml#L11).
* > for example Ivan's tests say that the RS needs to prompt you to start the MO playing; my belief is it shouldn't be automatic, but I think you can read chapter 1+2 here as the whole thing should be automatic.

The test currently says "While Chapter 1 is playing...", which does not make any assumptions on how playback is initiated or if it's manual or automatic (all this is therefore implementation-dependent - nowhere in the spec(s) do we say how an RS should initiate MO playback though it is generally a "play" button and whether "prompt you to start the MO playing" is the absolute best language to refer to a play button, is a rabbithole I am not going down :) Anyway my opinion is that it's fine as-is.

File: tests/mol-rendering-audio/EPUB/package.opf (hunk 6 above)
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-rendering-audio/EPUB/package.opf:14]: Sorry, missed this. Yes, that's correct.

**Comment by iherman** [comment]
@marisademeglio any reason why this PR should not be merged?

**Comment by marisademeglio** [comment]
> @marisademeglio any reason why this PR should not be merged?

No objection from me, I don't have any plans to make more changes.

**Comment by dlazin** [comment]
> > @marisademeglio any reason why this PR should not be merged?
> 
> Sorry, this is on me. I've repeatedly intended to re-review but kept not doing so. Lemme go get another coffee and then I shall! I'll commit once I'm happy. (And if I still want some changes, I'll just commit and send a followup PR, because I've kept Marisa waiting waaaaay too long on this one — sorry again!)

Ah, a bunch of the comments I left in the last review in February have not been addressed, but I will commit this PR anyway and then send a followup PR to make those changes.

**Comment by marisademeglio** [comment]
> > > @marisademeglio any reason why this PR should not be merged?
> > 
> > 
> > Sorry, this is on me. I've repeatedly intended to re-review but kept not doing so. Lemme go get another coffee and then I shall! I'll commit once I'm happy. (And if I still want some changes, I'll just commit and send a followup PR, because I've kept Marisa waiting waaaaay too long on this one — sorry again!)
> 
> Ah, a bunch of the comments I left in the last review in February have not been addressed, but I will commit this PR anyway and then send a followup PR to make those changes.

The decision as I understood it was to [not use these rendering-audio tests](https://github.com/w3c/epub-tests/pull/117#issuecomment-1047748796) and use Ivan's instead, so I didn't bother to address what you brought up related to rendering-audio.

As for your comment about the navigation test, I did [reply already](https://github.com/w3c/epub-tests/pull/117#issuecomment-1086148135).

I think then the only thing to do is remove the rendering-audio tests from this PR? Looking back, I see Ivan mentioned that, and I ran out of time to take care of it.

File: tests/mol-navigation/EPUB/css/base.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-navigation/EPUB/css/base.css]: FYI, I changed this for the second test in https://github.com/w3c/epub-tests/pull/148 (but see below — maybe we are deleting that test?).

File: tests/mol-rendering-audio/EPUB/package.opf [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    <meta property="dcterms:modified">2022-02-10T00:00:00Z</meta>
+    <meta property="media:duration" refines="#smil-1">00:00:18.479</meta>
+      <meta property="media:duration">00:00:18.479</meta>
```
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-rendering-audio/EPUB/package.opf:17]: Fixed in https://github.com/w3c/epub-tests/pull/148.

File: tests/mol-rendering-audio/EPUB/clip_times_tests.xhtml [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8"/>
+        <title>EPUB Media Overlays Tests: Rendering Audio</title>
+        <link rel="stylesheet" type="text/css" href="css/base.css"/>
+    </head>
+    <body id="body">
+        <h1>EPUB Media Overlays Tests: Rendering Audio</h1>
+        <p>Each sentence below should play in full when this page is played.</p>
```
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-rendering-audio/EPUB/clip_times_tests.xhtml:9]: Fixed in https://github.com/w3c/epub-tests/pull/148.

File: tests/mol-rendering-audio/EPUB/package.opf [hunk 9]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System implements Media Overlays' RS Spec "Rendering Audio" section.</dc:description>
```
[the remark this answers is not in the document]
Comment (dlazin) [reply tests/mol-rendering-audio/EPUB/package.opf:7]: Fixed in https://github.com/w3c/epub-tests/pull/148.

File: tests/mol-navigation/EPUB/ch2.xhtml (hunk 4 above)
Comment (dlazin) [reply tests/mol-navigation/EPUB/ch2.xhtml]: (Marisa did this.)

**Comment by dlazin** [comment]
> > > > @marisademeglio any reason why this PR should not be merged?
> > > 
> > > 
> > > Sorry, this is on me. I've repeatedly intended to re-review but kept not doing so. Lemme go get another coffee and then I shall! I'll commit once I'm happy. (And if I still want some changes, I'll just commit and send a followup PR, because I've kept Marisa waiting waaaaay too long on this one — sorry again!)
> > 
> > 
> > Ah, a bunch of the comments I left in the last review in February have not been addressed, but I will commit this PR anyway and then send a followup PR to make those changes.
> 
> The decision as I understood it was to [not use these rendering-audio tests](https://github.com/w3c/epub-tests/pull/117#issuecomment-1047748796) and use Ivan's instead, so I didn't bother to address what you brought up related to rendering-audio.
> 
> As for your comment about the navigation test, I did [reply already](https://github.com/w3c/epub-tests/pull/117#issuecomment-1086148135).
> 
> I think then the only thing to do is remove the rendering-audio tests from this PR? Looking back, I see Ivan mentioned that, and I ran out of time to take care of it.

A-ha! I finally understand. Sorry, I hadn't read the comment threads carefully enough. I thought that *your* mol-rendering-audio was staying and Ivan's was already gone. Now I see it's the converse. I just cleaned up that test in https://github.com/w3c/epub-tests/pull/148, but I will now send another commit to remove it.
